app/models/category.rb

```diff
@@ -0,0 +1,5 @@
+class Category < ActiveRecord::Base
+  has_many :videos, -> { order("title")}
```

Comment:
You lack a space after title bracket, but this can also be written using a symbol - `-> { order(:title) }`